cicd/post_test_results.sh

```diff
@@ -10,45 +10,36 @@ if [[ -n $UUIDS ]]
 then
   # construct the comment message
   message="Test results are available in Ibutsu:"
-  for uuid in $UUIDS
-  do
-    message="${message}\nhttps://url.corp.redhat.com/ibutsu-runs-${uuid}"
-  done
+
   # if it is a GitHub PR
   if [[ -n $ghprbPullId ]]; then
 
     # set +e so that if this POST fails, the entire run will not fail
     set +e
 
-    # check if there's already a comment by InsightsDroid
-    last_comment=$(curl \
+    # retrieve latest commit
+    last_commit=$(curl \
       -X GET -H "Accept: application/vnd.github.v3+json" \
       -H "Authorization: token ${GITHUB_TOKEN}" \
       -H "Content-Type: application/json; charset=utf-8" \
-      ${GITHUB_API_URL}/repos/${ghprbGhRepository}/issues/${ghprbPullId}/comments | \
-      jq 'map(select(.user.login == "InsightsDroid"))[-1]')
+      ${GITHUB_API_URL}/repos/${ghprbGhRepository}/pulls/${ghprbPullId}/commits | \
```

Comment:
Is this the best way to obtain the commit you want to comment on? What if another commit is pushed to the PR while this Jenkins job is running? Since the app-sre jobs use the github pull request builder plugin, I think you might want to use the github pull request builder env vars:

From https://github.com/jenkinsci/ghprb-plugin/blob/master/README.md#creating-a-job -- these env vars should be available to the script:

```
${ghprbActualCommit} if you want to use the head of the pull request branch (e.g. refs/pull/4/head); or
${sha1}, to use GitHub's tentative merge of the compare and base branches (e.g. refs/pull/4/merge) if the PR can be automatically merged or the head of the pull request branch (e.g. refs/pull/4/head) if they can not be automatically merged.
```